tests/e2e/android-java/app/src/main/java/com/segment/generated/EveryOptionalType.java

```diff
@@ -0,0 +1,141 @@
+/**
+* This client was automatically generated by Segment Typewriter. ** Do Not Edit **
+*/
+package com.segment.generated;
+
+import java.util.*;
+import com.segment.analytics.Properties;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+public final class EveryOptionalType extends SEGSerializable {
+  private Properties properties;
+
+  
+  private EveryOptionalType(Properties properties) {
+    this.properties = properties;
+	}
+
+  protected Properties toProperties() {
+    return properties;
+	}
+  
+
+  /**
+  * Builder for {@link EveryOptionalType}
+  */
+  public static class Builder {
+    private Properties properties;
+
+    /**
+    * Builder for {@link EveryOptionalType}
+    */
+    public Builder() {
+      properties = new Properties();
+    }
+
+
+    /**
+     * Optional any property
+     */
+    public Builder optionalAny(final @Nullable Object optionalAny) {
+      properties.putValue("optional any", optionalAny);
+      return this;
+    } 
+    
+
+    /**
+     * Optional array property
+     */
+    public Builder optionalArray(final @Nullable List<Object> optionalArray) {
+      List<?> p = TypewriterUtils.serialize(optionalArray);
+      properties.putValue("optional array", p);
+      return this;
+    } 
+    
+
+    /**
+     * Optional array with properties
+     */
+    public Builder optionalArrayWithProperties(final @Nullable List<OptionalArrayWithPropertiesItem1> optionalArrayWithProperties) {
+      List<?> p = TypewriterUtils.serialize(optionalArrayWithProperties);
+      properties.putValue("optional array with properties", p);
+      return this;
+    } 
+    
+
+    /**
+     * Optional boolean property
+     */
+    public Builder optionalBoolean(final @Nullable Boolean optionalBoolean) {
+      properties.putValue("optional boolean", optionalBoolean);
+      return this;
+    } 
+    
+
+    /**
+     * Optional integer property
+     */
+    public Builder optionalInt(final @Nullable Long optionalInt) {
+      properties.putValue("optional int", optionalInt);
+      return this;
+    } 
+    
+
+    /**
+     * Optional number property
+     */
+    public Builder optionalNumber(final @Nullable Double optionalNumber) {
+      properties.putValue("optional number", optionalNumber);
+      return this;
+    } 
+    
+
+    /**
+     * Optional object property
+     */
+    public Builder optionalObject(final @Nullable Object optionalObject) {
+      properties.putValue("optional object", optionalObject);
+      return this;
+    } 
+    
+
+    /**
+     * Optional object with properties
+     */
+    public Builder optionalObjectWithProperties(final @Nullable OptionalObjectWithProperties1 optionalObjectWithProperties) {
+      if(optionalObjectWithProperties instanceof SEGSerializable){
```

Comment:
Is this just a null check? If so, let's just compare against `null` to make it a bit more clear.